manim/mobject/text/text_mobject.py

```diff
@@ -730,7 +723,7 @@ def _get_settings_from_t2xs(
         return settings
 
     def _get_settings_from_gradient(
-        self, default_args: dict[str, Iterable[str]]
+        self, default_args: dict[str, str]
```

Comment:
I suggest that the values of `default_args` are typed as `Any`, because they're the arguments passed to `TextSetting`, a class from ManimPango. See its constructor [here](https://github.com/ManimCommunity/ManimPango/blob/9bfbaa71b3b5ae4309077d0c364bc04488b8b068/manimpango/cmanimpango.pyx#L11-L22).

```suggestion
        self, default_args: dict[str, Any]
```